CocoaHeadsNL/CocoaHeadsNL/AppDelegate.swift

```diff
@@ -40,9 +71,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         CKContainer.default().add(badgeResetOperation)
     }
 
-    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
-
-    }
+    // MARK: Notifications
+    
+//    func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
```

Comment:
This is not needed anymore since you are using iOS 10 style notifications. Let's remove this then.

Reply:
It is needed if you want to "receive" push notifications from other sources than cloud kit ;)